.github/workflows/release-tag.yaml

```diff
@@ -57,11 +57,12 @@ jobs:
           token: ${{ secrets.MESOSPHERECI_USER_TOKEN }}
           add-paths: preprovisioned/kib
           body: bumps KIB to latest version
-          commit-message: "fix: bump kib to ${{ github.ref }}"
+          commit-message: "fix: bump kib to ${{ github.ref_name }}"
           path: cluster-api-provider-preprovisioned
-          title: "fix: bump kib to ${{ github.ref }}"
+          title: "fix: bump kib to ${{ github.ref_name }}"
 
   sign-binary:
+    needs: release-artifacts
```

Comment:
```suggestion
    needs: upload-release-artifacts
```

Reply:
The job name is `release-artifacts`, you wouldn't want the suggested change.